### PR TITLE
Improves the security in ImagePicker

### PR DESCRIPTION
**Pull request by RamonGilabert**
We got some crashes because we were using ! a lot. Now we are not using it anywhere, pretty much.

BONUS:

- Now it compiles again in the simulator, it wasn't compiling the project.
- Deletes unneeded icons.
- Now the fetching is the correct one and is a bit smarter.

### CODE REVIEW

File: Source/ImagePickerController.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class ImagePickerController: UIViewController {
 
   lazy public var galleryView: ImageGalleryView = { [unowned self] in
     let galleryView = ImageGalleryView()
-    galleryView.delegate = self
+    //galleryView.delegate = self
```
Comment (zenangst) [inline Source/ImagePickerController.swift]: @RamonGilabert what's going on here?

Comment (RamonGilabert) [inline Source/ImagePickerController.swift]: OH FUCK.

**Comment by RamonGilabert** [comment]
@zenangst fixed.

**Comment by zenangst** [comment]
@RamonGilabert :cactus:

**Comment by richardoti** [comment]
Didn't have time to comment, spotted this PR only now. Will apply changes in a new PR then.

**Pull request merged**